src/panel-app/apps.component.js

```diff
@@ -38,18 +52,46 @@ export default function Apps(props) {
 }
 
 function sortApps(apps) {
-  return [...apps].sort((a, b) => {
-    const nameA = a.name.toUpperCase(); // ignore upper and lowercase
-    const nameB = b.name.toUpperCase(); // ignore upper and lowercase
-    if (nameA < nameB) {
-      return -1;
-    }
-    if (nameA > nameB) {
-      return 1;
-    }
+  const statusNumMap = {
+    MOUNTED: 2
+  };
+  return [...apps]
+    .sort((a, b) => {
+      const nameA = a.name.toUpperCase(); // ignore upper and lowercase
+      const nameB = b.name.toUpperCase(); // ignore upper and lowercase
+      if (nameA < nameB) {
+        return -1;
+      }
+      if (nameA > nameB) {
+        return 1;
+      }
+      // names must be equal
+      return 0;
+    })
+    .sort((a, b) => {
+      const statusA = statusNumMap[a.status] || 0;
+      const statusB = statusNumMap[b.status] || 0;
+      if (statusA > statusB) {
+        return -1;
+      } else if (statusA < statusB) {
+        return 1;
+      } else {
+        return 0;
+      }
+    });
```

Comment:
Sort alphabetically first, then by statuses. This results in a sort by status (mounted vs everything else) that is alphabetical within the group.

Reply:
statii